app/views/transactions/create_transfer.js.erb

```diff
@@ -23,5 +23,5 @@
     $('#transfer_exchange_rate').parents('#rate_col').removeClass('hidden');
   <% end %>
 <% end %>
-
-addTranferFormMasks();
+$('#submit_btn').prop('disabled', false);
```

Comment:
@AlexanderUlitin Submit button is not inside form, we use modal and custom #submit_btn for all transactions/transfears forms